documented-extensions.md

```diff
@@ -31,7 +31,7 @@ Support for any extension is OPTIONAL. When an extension definition uses
 MAY), this usage only applies to events that use the extension.
 
 Extensions always follow a common placement strategy for in-memory formats (e.g.
-[JSON](json-format.md), XML, Protobuffer) that are decided by those
+[JSON](json-format.md), XML, [Protocol Buffers(protobuf-format.md)) that are decided by those
```

Comment:
Missing a `]`

Reply:
@n3wscott are you ok with this change to "Protobuffer" ?

Reply:
> Missing a ]

Oops, so much for fixing typos ;) Updated it.

>  "Protobuffer"

"Protocol Buffers" is the official name as per the [project page](https://developers.google.com/protocol-buffers/). I've seen "protobuf" [for short](https://github.com/protocolbuffers/protobuf), but not really "Protobuffer".

Reply:
Yep - I just wanted a googler's +1 :-)

Reply:
protobuf or Protocol Buffers. also needs a `]`